wdl/methods/phasing/Helper.wdl

```diff
@@ -378,6 +378,7 @@ task Shapeit4 {
         String prefix
         Int num_threads
         Int memory
+        Float ps
```

Comment:
Needs a more verbose name, but even better would be a generic catch-all `extra_args` argument that could take any non-input command-line arguments.

Reply:
added!